site/guide/model-validation/_assess-compliance-assess.qmd

```diff
@@ -1,3 +1,37 @@
+:::: {.content-visible unless-format="revealjs"}
+1. In the left sidebar, click **Model Inventory**.
+
+1. Select a model by clicking on it or find your model by applying a filter or searching for it.^[[Working with the model inventory](/guide/model-inventory/working-with-model-inventory.qmd#search-filter-and-sort-models)]
+
+1. In the left sidebar that appears for your model, click **Validation Report** and then locate **2. Validation**.
+
+   You can now expand any subsection of the validation report you would like to work with.
+
+   For example: Select **2.1.1. Assumptions**.
+
+1. In any section of the documentation where the **Assessment** dropdown menu is available, select one of the available options:
+
+   - No compliance — No tests have been run and no other evidence has been provided
```

Comment:
Do these options need to be bolded since you need to select from them? Elsewhere, you bolded all the tab names that a user can choose from.

Reply:
I thought about that, and decided against it but you're right, let's bold them.